R/read_extract_acute.R

```diff
@@ -149,6 +149,7 @@ read_extract_acute <- function(year, file_path = get_boxi_extract_path(year = ye
       ipdc = "Inpatient Day Case Identifier Code",
       cij_ipdc = "CIJ Inpatient Day Case Identifier Code (01)",
       lineno = "Line Number (01)",
+      case_reference_number = "Case Reference Number [C]",
```

Comment:
Need to read in case ref no above this

Reply:
"Case Reference Number [C]" is missing from the boxi extract? where does this come from?

Reply:
nm. I was running a differnt year to you. Ran with 1920 and it works fine